config/sidebar.config.js

```diff
@@ -6,9 +6,9 @@ module.exports = {
         "concepts/understanding-hash-types",
         "concepts/deploy-and-deploy-lifecycle",
         //"concepts/global-state", // NEW CONTENT WILL BE HERE
+        "concepts/session-code",
```

Comment:
This is part of a series of PRs which all update this TOC. In my opinion we could have put them all together and it would have made more sense..... So which should come first ? Contracts or session code?

Reply:
This page introduces "session code" and compares it to the contracts, which IMO should be earlier - exactly like defined in the new structure.

Reply:
OK, but still commented, correct?